site/app/libraries/routers/Enabled.php

```diff
@@ -17,7 +17,7 @@
  * Example Usage:
  *
  * ```php
- * @Enabled("forum")
+ * @ Enabled("forum")
```

Comment:
We can actually get rid of the space here. Doctrine annotations should be okay with it here since it has a parameter for the constructor. Since this is technically a usage example I think it should be as accurate as possible. And of course the same for FeatureFlag.php.